app/presenters/menu/item.rb

```diff
@@ -1,5 +1,15 @@
 module Menu
   Item = Struct.new(:id, :name, :feature, :rbac_feature, :href, :type) do
+    extend ActiveModel::Naming
+
+    def self.base_class
+      Menu::Item
+    end
+
+    def self.base_model
+      model_name
+    end
+
```

Comment:
I agree, but maybe in a different PR? Changing these menu classes much more seems out of scope.